br.com.jeanhertel.adriconf.appdata.xml

```diff
@@ -0,0 +1,105 @@
+      <p>A GUI Tool used to configure OpenGL drivers. You can use it to optimize game settings or even workaround issues with them.</p>
+   </description>
+   <launchable type="desktop-id">br.com.jeanhertel.adriconf.desktop</launchable>
+   <screenshots>
+      <screenshot type="default">
+         <caption>Performance Window</caption>
+         <image type="source">https://github.com/jlHertel/adriconf/blob/master/flatpak/Screenshot0.png?raw=true</image>
+      </screenshot>
+      <screenshot type="default">
+         <caption>Debugging Window</caption>
+         <image type="source">https://github.com/jlHertel/adriconf/blob/master/flatpak/Screenshot1.png?raw=true</image>
+      </screenshot>
+   </screenshots>
+   <releases>
+      <release version="1.2" date="2018-09-06" urgency="low">
+         <description>
+            <p>This release has the following improvements.</p>
+            <ul>
+               <li>Improve README and use the oficial GPL notice by @Calinou</li>
+               <li>Fix minor issues with the debian package build</li>
+               <li>Add an icon and desktop entry file by @velurimithun</li>
+               <li>Add initial Flatpak support by @velurimithun</li>
+            </ul>
+         </description>
+      </release>
+      <release version="1.1" date="2018-08-05">
+         <description>
+            <ul>
+               <li>Add shorcuts for the common action in the menu (save, quit, new app, etc..)</li>
+               <li>Properly link against GLVND-enabled systems</li>
+               <li>Make GTKmm mandatory</li>
+               <li>Make Boost.locale mandatory</li>
+               <li>Explicitly declare dependencies of glib-compile-resources</li>
+               <li>Add various test cases</li>
+               <li>Properly handle closed-source drivers</li>
+            </ul>
+         </description>
+      </release>
+      <release version="1.0" date="2018-04-10">
+         <description>
+            <p>This is the first stable release.</p>
+         </description>
+      </release>
+      <release version="0.1-beta.1" date="2018-03-21">
+         <description>
+            <p>First oficial release of the software.</p>
+            <ul>
+               <li>This is still an alpha version and crashes are expected!</li>
+            </ul>
+         </description>
+      </release>
+      <release version="0.1-alpha" date="2018-04-04">
+         <description>
+            <ul>
+               <li>Don't build units tests by default</li>
+               <li>Fix installation of translations</li>
+               <li>Added more unit tests</li>
+               <li>Some cleanups to CMakeLists.txt to make it more readable</li>
+            </ul>
+         </description>
+      </release>
+   </releases>
+   <content_rating type="oars-1.1">
```

Comment:
This can be simplified: https://hughsie.github.io/oars/generate.html

Reply:
done